src/shared_modules/keystore/include/keyStore.hpp

```diff
@@ -14,23 +14,52 @@
 
 #include <string>
 
+#include "loggerHelper.h"
+#include "opensslWrapper.hpp"
 #include "rocksDBWrapper.hpp"
 
 constexpr auto DATABASE_PATH {"queue/keystore"};
 constexpr auto PRIVATE_KEY_FILE {"etc/sslmanager.key"};
 constexpr auto CERTIFICATE_FILE {"etc/sslmanager.cert"};
 
+constexpr auto KS_NAME {"keystore"};
```

Comment:
No sería mejor KEYSTORE_NAME?

Reply:
this code was imported from eliminated file `rsaHelper.hpp` and was reviewed in that time, But if you consider this change appropriate, I will proceed.

Reply:
When creating the constant I used the same terminology as the `indexer-connector`
```cpp
constexpr auto IC_NAME {"indexer-connector"};
```

Reply:
Let's keep it then. I was just curious.